tedana/workflows/tedana.py

```diff
@@ -221,6 +226,9 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
         Generate intermediate and additional files. Default is False.
     png : obj:'bool', optional
         Generate simple plots and figures. Default is false.
+    png-cmap : obj:'str', optional
```

Comment:
png_cmap

Reply:
I was matching --out-dir with that choice. I do prefer the underscore however.

Reply:
Sorry, I just mean for the function's docstring. The CLI argument is `--png-cmap`, but the Python function's argument is `png_cmap`, so the docstring for the function should have `png_cmap`.

Reply:
ah, i missed that. I'll go ahead and update that